webrender/src/gpu_cache.rs

```diff
@@ -468,9 +468,8 @@ impl<'a> GpuDataRequest<'a> {
         self.texture.pending_blocks.extend_from_slice(blocks);
     }
 
-    /// Consume the request and return the number of blocks written
-    pub fn close(self) -> usize {
```

Comment:
I will keep the close() function and create a new one for regular block num checking.